src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,108 @@
+@import "/styles/utils/reset.scss";
+@import "/styles/utils/_variables.scss";
+@import "/styles/blocks/stars.scss";
+
+.catalog {
+  width: $catalog-width;
+  display: flex;
+  flex-wrap: wrap;
+  padding: 48px 38px;
+  margin: 0 auto;
+}
+
+.container {
+  width: $card-width;
+  margin-right: $card-margin;
```

Comment:
you have margin-right 50px, its supposed to be 48px. You can see in chrome with dev tools? that container using 50, that's one of the reasons why you can't pass the test.